### PR TITLE
feat!: api swagger changes for bulk mvp

### PR DESCRIPTION
**Pull request by kirgene**
refactor bulkTransaction types

### CODE REVIEW

File: docs/sdk-scheme-adapter-outbound-v2_0_0-openapi3-snippets.yaml [hunk 1]
```diff
@@ -4694,7 +4696,7 @@ components:
       properties: *ref_40
     bulkTransferErrorResponse:
       allOf: *ref_92
-    individualTransactionAccept:
+    individualTransferAccept:
```
Comment (mdebarros) [inline docs/sdk-scheme-adapter-outbound-v2_0_0-openapi3-snippets.yaml:4699]: Should this not be `bulkTransactionIndividualTransferAccept`?

File: src/sdk-scheme-adapter/outbound/v2_0_0/json-schemas.json [hunk 2]
```diff
@@ -9172,7 +9172,7 @@
       }
     ]
   },
-  "individualTransactionAccept": {
+  "individualTransferAccept": {
```
Comment (mdebarros) [inline src/sdk-scheme-adapter/outbound/v2_0_0/json-schemas.json:9175]: Should it not be `BulkTransactionIndividualTransferAccept`?

File: src/sdk-scheme-adapter/outbound/v2_0_0/openapi.ts [hunk 3]
```diff
@@ -35697,11 +35693,9 @@ export interface components {
           };
         }[];
       };
-    } & {
-      bulkTansferState: unknown;
-    });
+    };
     /** @description Data model for the 'individualTransfer' while accepting party or quote. */
-    individualTransactionAccept: {
+    individualTransferAccept: {
```
Comment (mdebarros) [inline src/sdk-scheme-adapter/outbound/v2_0_0/openapi.ts:35698]: Should it not be `BulkTransactionIndividualTransferAccept`?

**Comment by kleyow** [comment]
lgtm

**Review by kleyow — APPROVED** [review]

**Review by mdebarros — APPROVED** [review]
+1

**Pull request merged**